app/controllers/bills_controller.rb

```diff
@@ -54,12 +54,43 @@ def show
     @user = current_user
     @household = Household.find(params[:household_id])
     @bill = Bill.find(params[:id])
+    @bills = Bill.where(date: @bill.date, household_id: @bill.household_id)
+    if @bills.length == 1
+      flash[:notice] = "Please add more bills for this month to see breakdown"
+    end
+    d = []
+    @bills.each do |bill|
+      d << [bill.kind, bill.amount]
+    end
     @data = Utilitydata.where(state: @household.state, kind: @bill.kind,
                               date: @bill.date).first
     if @data == nil
       @data = Utilitydata.new(state: @household.state, kind: @bill.kind,
                               date: @bill.date, amount: 10000000)
     end
+    text = "Breakdown for #{@bill.date.month}/#{@bill.date.year}"
+    @chart3 = LazyHighCharts::HighChart.new('pie') do |f|
+      f.chart({ defaultSeriesType:"pie", margin: [50, 200, 60, 170] } )
+      series = {
+        type: 'pie',
+        name: 'Amount($)',
+        data: d
+      }
+      f.series(series)
+      f.options[:title][:text] = text
+      f.legend(layout: 'vertical', style: { left: 'auto', bottom: 'auto', right: '50px', top: '100px' })
+      f.plot_options(pie:{
+        allowPointSelect:true,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Space missing after colon.